ui/app/appointments/controllers/manage/calendar/appointmentsCalendarViewController.js

```diff
@@ -23,7 +23,7 @@ angular.module('bahmni.appointments')
                 event.color = appointment.service.color;
                 event.serviceName = appointment.service.name;
                 var existingEvent = _.find(eventList, event);
-                var patientName = appointment.patient.name + " (" + appointment.patient.identifier + ")";
```

Comment:
@prashantsable1991 Instead of splitting the name after it has been united, why not update it in the service when the united name is created?